### PR TITLE
add tools to drop privileges if running test under root

### PR DESCRIPTION
**Pull request by paulperegud**
For details see:
https://github.com/saleyn/erlexec/blob/41ded9e24b055c8176b4fb5c7cfad7a9a7213d93/src/exec.erl#L630
and
http://saleyn.github.io/erlexec/

### CODE REVIEW

**Comment by paulperegud** [comment]
@sirn @pdobacz 

`erlexec` refuses to run external programs with root rights. This is the workaround.

**Comment by paulperegud** [comment]
blocks https://github.com/omisego/omisego/pull/60

**Review by sirn — APPROVED** [review]

**Review by pdobacz — APPROVED** [review]

**Comment by paulperegud** [comment]
@sirn I can't merge this one myself, no write access.

**Pull request merged**
